cql3/operation.cc

```diff
@@ -18,6 +18,7 @@
 #include "types/list.hh"
 #include "types/set.hh"
 #include "types/user.hh"
+#include "raft/broadcast_tables/experimental/lang.hh"
```

Comment:
This should not be under raft/. It's using raft but it's a lot more cql and database than raft.

It's okay to move it post-merge.

Reply:
Maybe `service/raft/broadcast_tables/`

Reply:
> It absolutely shouldn't.

Right, I will move it.

> How hard is it to move them now?

I think it won't be hard.

Reply:
In that case, the namespaces should be fixed too. If you want to move it under service/, raft::broadcast_tables -> service::raft::broadcast_tables.

Reply:
But it will pull cql into group0 code. I'd be much happier if it was in a different group. As it is, whereever we put it it will entangle two unrelated things, so we might as well keep it here.

Reply:
Raft is an independent library. It is fine to entangle it with group0 while it uses group0, but not with entire raft. BTW `service/raft/` contains not only group0 code. Generic raft group management is also there. We can also put it into `service/broadcast_tables/` and pretend that raft usage is just an implementation detail.

Reply:
> But it will pull cql into group0 code

Will it? broadcast tables module has no CQL specific code. The CQL specific parts live in cql3/statements/strongly_consistent_* files.

Reply:
ok